src/transformers/models/yolos/configuration_yolos.py

```diff
@@ -74,8 +74,8 @@ class YolosConfig(PretrainedConfig):
             The number of detection tokens.
         use_mid_position_embeddings (`bool`, *optional*, defaults to `True`):
             Whether to use the mid-layer position encodings.
-        auxiliary_loss (`bool`, *optional*, defaults to `False`):
-            Whether auxiliary decoding losses (loss at each decoder layer) are to be used.
```

Comment:
@amyeroberts I just added one caution for auxiliary_loss usage in YOLOS. Feel free to modify!

Reply:
Ah, I think I should clarify a bit: we should try to avoid changing param names, values and types as much as possible in config files. So in this can, we should keep `auxiliary_loss` here. 

It's true we'd like the arguments to be more aligned, but we have to balance that with backwards compatibility: the user should be able to load in an old config and expect the same behaviour. In this case, modifying the name requires additional logic to properly handle this, which is outside the scope of this PR.

Reply:
@amyeroberts Yeap, I will revert the variable name to 'auxiliary_loss' but keep the additional description!